src/main/scala/is/hail/HailContext.scala

```diff
@@ -178,56 +180,59 @@ object HailContext {
     ProgressBarBuilder.build(sc)
   }
 
-  def readRowsPartition(t: TStruct, codecSpec: CodecSpec)(i: Int, in: InputStream, metrics: InputMetrics = null): Iterator[RegionValue] = {
-    new Iterator[RegionValue] {
-      private val region = Region()
-      private val rv = RegionValue(region)
-
-      private val trackedIn = new ByteTrackingInputStream(in)
-      private val dec =
-        try {
-          codecSpec.buildDecoder(trackedIn)
-        } catch {
-          case e: Exception =>
-            in.close()
-            throw e
-        }
+  def readRowsPartition(
+    t: TStruct,
+    codecSpec: CodecSpec
+  )(ctx: RVDContext,
+    in: InputStream,
+    metrics: InputMetrics = null
+  ): Iterator[RegionValue] = new Iterator[RegionValue] {
+    private val region = ctx.region
+    private val rv = RegionValue(region)
+
+    private val trackedIn = new ByteTrackingInputStream(in)
+    private val dec =
+      try {
+        codecSpec.buildDecoder(trackedIn)
+      } catch {
+        case e: Exception =>
+          in.close()
+          throw e
+      }
 
-      private var cont: Byte = dec.readByte()
-      if (cont == 0)
-        dec.close()
+    private var cont: Byte = dec.readByte()
+    if (cont == 0)
+      dec.close()
```

Comment:
Esp. when the change of formatting induces a change of indentation which causes the diff to explode and makes it much harder to tell what's change.

My advice would be to leave the formatting for existing functions (even if you add/remove arguments) in the style we had before.  It's fine to use the newer formatting for new things you add.